vendor/bitcoin_support/src/network.rs

```diff
@@ -0,0 +1,30 @@
+use bitcoin::network::constants::Network as ExternNetwork;
```

Comment:
:x: I'd prefer if you don't call this `ExternNetwork` but just write the `From`-impl with the FQN.

Reply:
I also always prefer that. I don't know why I did it like this lol.